easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.12-GCC-10.2.0.eb

```diff
@@ -30,4 +30,11 @@ checksums = [
 # extensive testing can be enabled by uncommenting the line below
 # runtest = 'PATH=.:$PATH lapack-timing'
 
+# enable iterative build with 64-bit integer support (as well as default 32-bit)
+build_ilp64 = True
+# library name suffix to append to 64-bit versions
+ilp64_lib_suffix = 'ilp64'
+# symbol suffix to append to 64-bit versions
+ilp64_symbol_suffix = '64_'
```

Comment:
Should suffix really be "64_" and not "_64" ??

Reply:
I think you're both right :) https://github.com/xianyi/OpenBLAS/issues/646#issuecomment-167888870

Reply:
Just to add to this as I am trying to get an EC for GAMESS-US, latest version, going. The do this:
```
: openblas
Where is your libopenblas64.a or libopenblas.a file located?
```
Shall we patch that for that GAMESS-US version or consider doing yet another symlink here?

Reply:
Update on that: My older manual `OpenBLAS` installtion has these files:
```
libopenblas_64bit.so 
libopenblas_64bit.a
```